### ISSUE TITLE
Usersnap Feedback - I think I may have asked this a couple of years ago, but why do we make it [...]

### ISSUE THREAD

**Issue by ascott** [participedia/usersnaps#1249]
**Sender**: pscully@clearviewconsultingllc.com
**Comment**: I think I may have asked this a couple of years ago, but why do we make it so difficult for people to upload images to their profile? Or maybe I am alone in thinking this is a pain in the neck.  I wanted to test the system of what it looks like to create a new account, but once I got to the Avatar page I didn't want to take the time to figure out another system.
[Open #938 in Usersnap Dashboard](https://usersnap.com/a/#/participedia-dev/p/participedia-net-9cb6d041/938)

<a href='https://usersnappublic.s3.amazonaws.com/2020-11-02/23-55/a97c21e5-b7f6-488b-955f-8103936a372e.png'>![Usersnap Feedback - I think I may have asked this a couple of years ago, but why do we make it (...)](https://usersnappublic.s3.amazonaws.com/2020-11-02/23-55/aa67b3fc-f449-4887-ba0d-724ca4bafaf3.jpg)</a>
<a href='https://usersnappublic.s3.amazonaws.com/2020-11-02/23-55/a97c21e5-b7f6-488b-955f-8103936a372e.png'>Download original image</a>

**Browser**: Firefox 82 (Windows 10)
**Referer**: [https://participedia.net/user/419319/edit](https://participedia.net/user/419319/edit)
**Screen size**: 1760 x 1100 **Browser size**: 1760 x 958

Powered by [usersnap.com](https://usersnap.com/?utm_source=github_entry&utm_medium=web&utm_campaign=product)

**Comment by jesicarson** [comment]
@plscully Good point, I'll definitely add it to our list of possible projects to undertake. We are in the process of evaluating what we can take on in the transition to phase 2. In terms of why we use gravatar, it was a time-saving trade off between using a 3rd party system or building our own (similar to why we use auth0 instead of building a bespoke authorization system of our own). There may be another system that works better than gravatar, but the research for that has not yet been re-prioritized. With such a small team, it's always a process of weighing the value of a new feature for users compared to the dev time, and of course comparing that to all the other priorities we could work on. We could also look at analytics for user profiles, and I'm working on a user survey that will ask folks what features they find most useful (user profiles are on that list).

**Comment by plscully** [comment]
@jesicarson Thank you! I realize that you all are doing a lot with very little time budgeted to get everything done. I just wanted to make sure this was on your list.

**Comment by jesicarson** [comment]
100% 
I agree the current process is not ideal.
